src/k2/cmd/tpcc/tpcc_client.cpp

```diff
@@ -297,9 +304,11 @@ class Client {
             auto cntpsec = (double)_completedTxns/totalsecs;
             auto readpsec = (double)_client.read_ops/totalsecs;
             auto writepsec = (double)_client.write_ops/totalsecs;
+            auto querypsec = (double)_client.query_ops/totalsecs;
```

Comment:
Query requests actually are reads, but i am not sure how to combine it with read counts is reasonable. So at present query requests are counted separately.

Reply:
I agree, counting the queries separately is good.